### DISCUSSION TITLE
Scrollable TextBox

### DISCUSSION THREAD

**Discussion by HowardPWeiss** [unoplatform/uno#13038] [Q&A]
I building a C# application for Linux using VisualStudioCode and UnoPlatform

I would like to have a scrollable read only text box which I can use to present a history.  At each event, I would add more text to the bottom of the text box and scrolll that text into view.  Any suggestions?

In a windows form based application, I used a rich text box for this functionality

**Comment by limeniye** [comment]
I'am not sure, but I think you looking for [ISupportIncrementalLoading](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.data.isupportincrementalloading?view=winrt-22621). 
I saw something by @nickrandolph  [here](https://nicksnettravels.builttoroam.com/incremental-loading-pagination-with-mvvm-and-mvux/)

And also I recommend you to check the following links [stackoverflow](https://stackoverflow.com/a/45208956), [marcominerva](https://marcominerva.wordpress.com/2013/05/22/implementing-the-isupportincrementalloading-interface-in-a-window-store-app/)

**Comment by FrozDark** [comment]
Put TextBlock inside ScrollViewer
